packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/ovirt/validateOvirtURL.ts

```diff
@@ -1,12 +1,19 @@
 import { validateURL, ValidationMsg } from '../../common';
 
 export const validateOvirtURL = (url: string | number): ValidationMsg => {
+  if (url === undefined) {
+    return {
+      type: 'default',
+      msg: 'The URL of the Red Hat Virtualization Manager (RHVM) API endpoint, for example: https://rhv-host-example.com/ovirt-engine/api .',
+    };
+  }
+
   // Sanity check
   if (typeof url !== 'string') {
     return { type: 'error', msg: 'URL is not a string' };
   }
 
-  const trimmedUrl: string = url.toString().trim();
+  const trimmedUrl: string = url?.toString().trim();
```

Comment:
we can't get here is url is null, check in line 4, we can't get here is url is not string, check in line 12
```suggestion
  const trimmedUrl: string = url.trim();
```